test/Collection.spec.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import { shallow, mount } from 'enzyme';
+import { mount } from 'enzyme';
```

Comment:
did we confirm that these specs fail with shallow? :)

Reply:
Yeah, it's fine with shallow, restored it.